backend/test/card_game/core_base_test.clj

```diff
@@ -0,0 +1,51 @@
+(ns card-game.core-base-test
+  (:require [expectations.clojure.test :refer :all]
+            [card-game.core :as core]
+            [card-game.victory-conditions :as victory-conditions]
+            [card-game.test-helper :as helper]))
+
+(defexpect basic.game
+  ; Game can be created
+  (expect
+    #(not (nil? %))
+    (core/new-game))
+
+  ; Game contains two players
+  (expect
+    #(count (:players %))
+    (core/new-game))
+
+  ; Hand's not empty
+  (expect
+    false
+    (-> (core/new-game)
+        (get-in [:players 0 :hand])
+        (empty?))))
+
+(defexpect points-on-rows
+  ; We can get the current amount of points on different rows
```

Comment:
These functions really look like they could be simplified... I don't know how right now, though.